### PR TITLE
chore(provider) basic sweeper implementation

### PR DESCRIPTION
**Pull request by nicolai86**
\cc @stack72. thanks for pointing me at the AWS provider, man!

```
go test . -v -sweep=par1
2017/06/19 19:18:55 [DEBUG] Running Sweepers for region (par1):
2017/06/19 19:18:55 [DEBUG] Destroying the volumes in (par1)
2017/06/19 19:18:55 [DEBUG] HEAD /volumes
2017/06/19 19:18:55 [DEBUG] [[HEAD https://cp-par1.scaleway.com/volumes?]]: %!v(MISSING)
2017/06/19 19:18:55 [DEBUG] GET /volumes
2017/06/19 19:18:55 [DEBUG] [[GET https://cp-par1.scaleway.com/volumes]]: %!v(MISSING)
2017/06/19 19:18:55 [DEBUG] [Response]: [[200 {"volumes": []}]]
%!v(MISSING)
2017/06/19 19:18:55 [DEBUG] Destroying the ips in (par1)
2017/06/19 19:18:55 [DEBUG] HEAD /ips
2017/06/19 19:18:55 [DEBUG] [[HEAD https://cp-par1.scaleway.com/ips?]]: %!v(MISSING)
2017/06/19 19:18:55 [DEBUG] GET /ips
2017/06/19 19:18:55 [DEBUG] [[GET https://cp-par1.scaleway.com/ips]]: %!v(MISSING)
2017/06/19 19:18:55 [DEBUG] [Response]: [[200 {"ips": []}]]
%!v(MISSING)
2017/06/19 19:18:55 [DEBUG] Destroying the security groups in (par1)
2017/06/19 19:18:55 [DEBUG] HEAD /security_groups
2017/06/19 19:18:55 [DEBUG] [[HEAD https://cp-par1.scaleway.com/security_groups?]]: %!v(MISSING)
2017/06/19 19:18:55 [DEBUG] GET /security_groups
2017/06/19 19:18:55 [DEBUG] [[GET https://cp-par1.scaleway.com/security_groups]]: %!v(MISSING)
2017/06/19 19:18:56 [DEBUG] [Response]: [[200 {"security_groups": [{"description": "Auto generated security group.", "enable_default_security": true, "servers": [], "organization": "7b4c2f27-067d-4a79-b721-1aaac4475889", "organization_default": true, "id": "4e89dc74-556e-41da-bafd-86bfecfc53f7", "name": "Default security group"}]}]]
%!v(MISSING)
2017/06/19 19:18:56 [DEBUG] Destroying the servers in (par1)
2017/06/19 19:18:56 [DEBUG] HEAD /servers
2017/06/19 19:18:56 [DEBUG] HEAD /servers
2017/06/19 19:18:56 [DEBUG] [[HEAD https://cp-ams1.scaleway.com/servers?]]: %!v(MISSING)
2017/06/19 19:18:56 [DEBUG] [[HEAD https://cp-par1.scaleway.com/servers?]]: %!v(MISSING)
2017/06/19 19:18:56 [DEBUG] GET /servers
2017/06/19 19:18:56 [DEBUG] [[GET https://cp-par1.scaleway.com/servers]]: %!v(MISSING)
2017/06/19 19:18:56 [DEBUG] [Response]: [[200 {"servers": []}]]
%!v(MISSING)
2017/06/19 19:18:57 [DEBUG] GET /servers
2017/06/19 19:18:57 [DEBUG] [[GET https://cp-ams1.scaleway.com/servers]]: %!v(MISSING)
2017/06/19 19:18:57 [DEBUG] [Response]: [[200 {"servers": []}]]
%!v(MISSING)
2017/06/19 19:18:57 Sweeper Tests ran:
	- scaleway_volume
	- scaleway_ip
	- scaleway_security_group
	- scaleway_server
ok  	github.com/terraform-providers/terraform-provider-scaleway/scaleway	2.828s
```

### CODE REVIEW

File: scaleway/resource_scaleway_volume_attachment.go [hunk 1]
```diff
@@ -10,6 +10,36 @@ import (
 	"github.com/scaleway/scaleway-cli/pkg/api"
 )
 
+func init() {
```
Comment (catsby) [inline scaleway/resource_scaleway_volume_attachment.go]: `init` and `testSweepVolume` need to be in `resource_scaleway_volume_attachment_test.go` file

**Comment by nicolai86** [comment]
I thought I had pushed that accidental placement change already... 🤔 will fix asap

**Comment by nicolai86** [comment]
thank you for your feedback @catsby ! very much appreciated!

**Comment by nicolai86** [comment]
🎉

**Pull request merged**